presto-spi/src/main/java/com/facebook/presto/spi/Page.java

```diff
@@ -252,6 +252,17 @@ private static int determinePositionCount(Block... blocks)
         return blocks[0].getPositionCount();
     }
 
+    public static Page mask(Page page, int[] retainedPositions)
```

Comment:
It's not, but will be in Spill (partitioning spiller and LookupJoinOperator both need this). Should I move adding this method to that PR?